autora/andressa-andressa/index.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+
+<html lang="pt-br">
+
+<head>
+    <link href="https://fonts.googleapis.com/css?family=Montserrat|Roboto" rel="stylesheet">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="css/style.css">
+
+    <title>Document</title>
+</head>
+
+
+<body>
+    <header class="cabeçalho">
+
+        <nav class="navigation">
+            <div class="navigation_maravilhosas">
+                <img src="img/logo-white.png" alt=" M de Maravilhosas" class="navigation_logo">
+            </div>
+
+            <ul class="navigation_lista">
+                <li><a href="../../index.html" class="links">Home</a></li>
+                <li><a href="#" class="links">Conheça a {Reprograma}</li>
+            </ul>
+
+            <div class="navigation_media">
+                <img src="img/icon-facebook.png" alt="Facebook">
+                <img src="img/icon-instagram.png" alt="Instagram">
+                <img src="img/icon-linkedin.png" alt="Linkedin">
+                <img src="img/icon-twitter.png" alt="Twitter">
+            </div>
+
+            <div class="menu_hamburger">
+                <img src="img/menu-nav.png">
+            </div>
+        </nav>
+        <h1 class="nome">Sobre a autora</h1>
+    </header>
+
+    <main>
+
+        <section class="conteudo">
+            <div class="resumo">
+                <div>
+                    <h1>Andressa Santos</h1>
+                </div>
+
+                <div class="resumo2">
+                    <div class="autora">
+                        <img src="img/andressa-andressa.jpg" class="autora_img">
+                    </div>
+
+                    <div class="sobre_autora">
+                        <p> Andressa Santos, Desenvolvedora Front-End, gosto de cachorros, chocolate, e de
+                            joguinhos, How I
+                            met your mother é melhor que Friends.
+                        </p>
+                        
+                        <p><span>Autora do artigo de: <a href="../../personalidade/rihanna-rihanna/index.html"><em>Rihanna</em></a></p>
+                        
+                            
+                        <ul>
+                            <li><a href="https://www.linkedin.com/in/andressa-santos-437ba7180/" target="_blank">Linkedin |</a></li>
+                            <li><a href="https://github.com/andrssasnts" target="_blank">GitHub |</a></li>
+                            <li><a href="https://www.instagram.com/andrssasnts" target="_blank"> Instagram </a></li>
+                        </ul>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <section class="conteudo2">
+            <div class="conteudo3">
```

Comment:
Aqui é apenas uma sugestão: quando utilizar classes, tente adicionar nomes que descreve melhor o que e onde você está modificando. Uma boa é usar o método BEM. Por exemplo, nessa class você colocou "conteudo3", seria interessante você escrever "conteudo__maravilhosas". Quando você for revisar seu código ou quando outras pessoas forem usar ou mexer no seu código ficará muito mais fácil de encontrar onde alterar. :)